storage/jellyfish-merkle/src/lib.rs

```diff
@@ -946,6 +775,174 @@ where
     }
 }
 
+/// Get the node hash from the cache if cache is provided, otherwise (for test only) compute it.
+fn get_hash<K>(
```

Comment:
@aptos-labs/storage -- these became functions (outside of the struct), instead of methods to appease this clippy: https://rust-lang.github.io/rust-clippy/master/#only_used_in_recursion. To be fair, I think it's clearer having these as functions because it immediately tells me they aren't modifying the struct state 😄